docs/root/intro/version_history.rst

```diff
@@ -1,6 +1,24 @@
 Version history
 ---------------
 
+0.3.0 (Mar 26,2020)
+===================
+
+This is the first release of Envoy Mobile we are tagging as production ready 🎉
```

Comment:
Would it be alright if we downplayed this a bit more? I don't feel comfortable advertising that this is production ready for the general audience.

Maybe something like:
```
This is the first release of Envoy Mobile we are using in a production application! :tada:
```

Reply:
Sure that sounds fine to me. Do you still feel comfortable tagging the release without the `pre-release` tag? We have done all of our due diligence internally and are using in prod, so I don't see a problem with tagging the release without it

Reply:
I'm fine with that :)

Reply:
I guess the main concern I have when we say that this is ready for production and in our next minor release, we have a patch for a very blaring issue we discovered in prod

Reply:
so it goes ;)